resources/packs/gcp/gcp.lr

```diff
@@ -581,7 +581,8 @@ private gcp.sql.instance @defaults("name") {
   // Instance state
   state string
   // Project
-  project string // TODO: should this be id or name
+  // TODO: should this be id or name
```

Comment:
This one is still showing up in the docs with this change

Reply:
hmm, maybe that uses a way that is different than the shell. let me look into it

Reply:
should be fixed now

Reply:
It's getting closer, but now the "Project" description is getting removed as well

Reply:
Working now